features/payment/managing_payment_methods/adding_payment_method.feature

```diff
@@ -8,22 +8,28 @@ Feature: Adding a new payment method
         Given the store is available in "English (United States)"
         And I am logged in as an administrator
 
-    @ui
+    @ui @javascript @current
     Scenario: Adding a new payment method
         Given I want to create a new payment method
         When I name it "Paypal Express Checkout" in "English (United States)"
         And I specify its code as "PEC"
         And I choose "Paypal Express Checkout" gateway
+        And I specify its "Username" as "test"
```

Comment:
@GSadee, different payment gateways may have different options, we can't foresee  what options it will be, in the case of  `Paypal Express Checkout` , it has Username, Password, etc, but another gateway might be different, it we don't try this way, we might need to create lots of methods , their only difference is just its method name.